packages/integration-tests/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "es2018",
+    "module": "commonjs",
+    "strict": true,
+    "esModuleInterop": true,
```

Comment:
Could this extend the other `tsconfig`?

Reply:
It could, but I'd rather do this once we move back to the monorepo.